controlplane/kubeadm/internal/cluster_test.go

```diff
@@ -439,13 +430,10 @@ func TestManagementCluster_healthCheck_Errors(t *testing.T) {
 				Client: &fakeClient{list: tt.machineList},
 			}
 			err := m.healthCheck(ctx, tt.check, clusterKey, controlPlaneName)
-			if err == nil {
-				t.Fatal("Expected an error")
-			}
+			g.Expect(err).To(HaveOccurred())
```

Comment:
If inlining the one above, might be best to inline this one too
```suggestion
			g.Expect(m.healthCheck(ctx, tt.check, clusterKey, controlPlaneName)).NotTo(Succeed())
```

Reply:
but we need to have the `err` below to validate the error

Reply:
Good point 🤦‍♂️ Friday afternoons...